Makefile

```diff
@@ -12,7 +12,4 @@ init: ## Initialize Terraform, on first-use or when adding modules or plugins.
 	@git config core.hooksPath .githooks
 
 format: ## Format your code automatically.
-	@terraform fmt
-
-plan: ## See how your changes would affect our infrastructure if applied.
-	@terraform plan
+	@terraform fmt -recursive
```

Comment:
The `-recursive` flag allows us to run this from the root level & automatically format any files in this repo!